src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/storage/RejectedFilesHandlerTest.java

```diff
@@ -37,7 +37,7 @@
     @BeforeEach
     void setUp() {
         dbHelper.deleteAll();
-        mover = new RejectedFilesHandler(storageClient, envelopeService);
+        mover = new RejectedFilesHandler(envelopeService, new BlobRejecter(storageClient));
```

Comment:
This class no longer accesses blob storage directly.
I'll remove integration test for it and introduce a new one (for `BlobRejecter`) in the next PR.